app/src/main/res/values/strings.xml

```diff
@@ -103,6 +104,35 @@
     <string name="nearby_detail_additional_information">Additional information</string>
     <string name="nearby_detail_owner">Owner</string>
     <string name="nearby_detail_contact">Contact</string>
-    <string name="item_quantity_unit">%1$s %2$s</string>
+    <string name="item_quantity_unit">%1$d %2$s</string>
+
+    <!-- Alert dialogs -->
+    <string name="ad_title_contact_email_missing">Contact email missing</string>
+    <string name="ad_title_contact_email_missing_on_destroy">Item not shared</string>
+    <string name="ad_title_error_parsing_address_string">Location Problem</string>
+    <string name="ad_title_no_network">No internet connection</string>
+    <string name="ad_title_invalid_address">Invalid address</string>
+    <string name="ad_title_save_item_failed">Item not saved</string>
+    <string name="ad_title_location_determination_failed">Unable to determine location</string>
+    <string name="ad_title_location_permission_rationale">Why do we need your location?</string>
+    <string name="ad_title_account_not_verified">Account not verified</string>
+    <string name="ad_title_location_permission_denied">Permission denied</string>
+
+    <string name="ad_msg_contact_email_missing">In order to share an Item, you have to provide a contact email other users can use to reach out to you. Please state your email address in the settings tab or disable the sharing option.</string>
+    <string name="ad_msg_contact_email_missing_on_destroy">In order to share an Item, you have to provide a contact email other users can use to reach out to you. Please state your email address in the settings tab.</string>
+    <string name="ad_msg_error_parsing_address_string">There was an error while we tried to match the location you\'ve entered to real coordinates. Please correct the address you\'ve entered or disable the sharing option. You may also want to check your internet connection and try again.</string>
+    <string name="ad_msg_error_parsing_address_string_on_destroy">There was an error while we tried to match the location you\'ve entered to real coordinates. Therefore, we disabled the sharing option. Please correct the address you\'ve entered in order to share the foodstuff. You may also want to check your internet connection and try again.</string>
+    <string name="ad_msg_no_network">There was an error while we tried to match the location coordinates to a real address. Therefore, we disabled the sharing option. Please check your internet connection and try again.</string>
```

Comment:
Why this again? Isn't this a duplicate of 50 and following?

Reply:
Oh, that must have been my mistake while resolving merge conflicts. I will fix that!